src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SpeechToTextImplementation.shared.cs

```diff
@@ -41,4 +85,42 @@ static async Task<bool> IsSpeechPermissionAuthorized()
 		return status is PermissionStatus.Granted;
 	}
 #endif
+}
+
+/// <summary>
+/// Event occurred on SpeechToText Recognition Result Updated
+/// </summary>
+public class OnSpeechToTextRecognitionResultUpdated : EventArgs
```

Comment:
Can those EventArgs be moved to another file?
Also, is a pattern on .NET to finish the name with `Args` to be clear that this model is an `EventArgs`, so consider changing it